.github/workflows/pr-validation.yaml

```diff
@@ -127,6 +127,23 @@ jobs:
       - name: Build and load (current arch)
         run: |
           docker buildx build --load -t ${{ env.NAME }}:${{ steps.prep.outputs.VERSION }} .
+      - name: Setup go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.16"
+      - name: Install libgit2
+        run: |
+          cd /tmp
+          git clone https://github.com/libgit2/libgit2.git
+          cd libgit2
+          git checkout maint/v1.1
+          mkdir build && cd build
+          cmake ..
+          sudo cmake --build . --target install
+          sudo ldconfig
+      - name: Run integration tests
+        run: |
+          make verify GITOPS_PROMOTION_IMAGE=${{ env.NAME }}:${{ steps.prep.outputs.VERSION }}
```

Comment:
Is there any risk that two tests can run in parallel? Do we support it?

Reply:
It is my understanding that tests need to say `t.Parallel()` to be run in parallel? Or do you mean if two PR pipelines are run at the same time?

Reply:
I was thinking about the PR pipelines

Reply:
I suppose the easiest way would be to have each test run set up a new app (presumably by copying a template) and gitops that app. Though they may still run into rate limits, particularly the weakly documented anti-burst "secondary" rate limits. It would require some thought, so with your leave I will defer that to a separate PR.